src/main/java/org/commcare/session/RemoteQuerySessionManager.java

```diff
@@ -162,13 +162,8 @@ public static String evalXpathExpression(XPathExpression expr,
     }
 
     public void populateItemSetChoices(QueryPrompt queryPrompt) {
-        EvaluationContext evalContextWithAnswers = evaluationContext.spawnWithCleanLifecycle();
-        Map<String, String> userQueryValues = getUserQueryValues(true);
-        userQueryValues.forEach((promptId, value) -> {
-            evalContextWithAnswers.setVariable(promptId, userAnswers.get(promptId));
-        });
```

Comment:
I think we'll need to preserve this for existing apps right?

Reply:
Jenny actually did [some investigation](https://dimagi-dev.atlassian.net/browse/USH-1776?focusedCommentId=211489) on this and found that no apps are currently using the existing variable reference syntax.